client/src/components/Search/Search.tsx

```diff
@@ -64,8 +64,8 @@ const Search = ({ search, handleChange }: Props): JSX.Element => {
         onClose={() => {
           setOpen(false);
         }}
-        getOptionSelected={(option, value) => option.username === value.username}
-        getOptionLabel={(option) => option.username}
+        // getOptionSelected={(option, value) => option.username === value.username}
```

Comment:
remove?

Reply:
The reason I do that is because recently in the standup, the team decided we don't need username for the application. Instead, we will ask for first name and last name. Currently, this Search component was a boilerplate code from the initial setup. Search functionality is yet to be done. This one is not working.